lib/disambiguate.js

```diff
@@ -8,14 +8,12 @@ function *disambiguate(heroku, pipeline_id_or_name) {
   if(validator.isUUID(pipeline_id_or_name)) {
     pipeline = yield heroku.request({
       method: 'GET',
-      path: `/pipelines/${pipeline_id_or_name}`,
-      headers: { 'Accept': 'application/vnd.heroku+json; version=3.pipelines' }
+      path: `/pipelines/${pipeline_id_or_name}`
```

Comment:
Do we have a new enough version of the heroku client that we can do this with `heroku.pipelines(identity).info()` yet?